osu.Game/Skinning/LegacyKeyCounter.cs

```diff
@@ -0,0 +1,114 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Sprites;
+using osu.Framework.Graphics.Textures;
+using osu.Game.Graphics;
+using osu.Game.Graphics.Sprites;
+using osu.Game.Screens.Play.HUD;
+
+namespace osu.Game.Skinning
+{
+    public partial class LegacyKeyCounter : KeyCounter
+    {
+        public bool UsesFixedAnchor { get; set; }
+
+        public float TransitionDuration { get; set; } = 50f;
+
+        public Colour4 KeyTextColour
+        {
+            get => keyTextColour;
+            set
+            {
+                keyTextColour = value;
+                overlayKeyText.Colour = value;
+            }
+        }
+
+        private Colour4 keyTextColour = Colour4.White;
+
+        private float keyTextRotation;
+
+        public float KeyTextRotation
+        {
+            get => keyTextRotation;
+            set
+            {
+                keyTextRotation = value;
+                overlayKeyText.Rotation = value;
+            }
+        }
+
+        public Colour4 KeyDownBackgroundColour = Colour4.Yellow;
+
+        private readonly Container keyContainer;
+
+        private readonly OsuSpriteText overlayKeyText;
+
+        private readonly Sprite keySprite;
+
+        public LegacyKeyCounter(InputTrigger trigger)
+            : base(trigger)
+        {
+            Origin = Anchor.Centre;
+            Anchor = Anchor.Centre;
+            Child = keyContainer = new Container
+            {
+                RelativeSizeAxes = Axes.Both,
+                Origin = Anchor.Centre,
+                Anchor = Anchor.Centre,
+                Children = new Drawable[]
+                {
+                    keySprite = new Sprite
+                    {
+                        Blending = BlendingParameters.Mixture,
```

Comment:
dunno where this even came from, as far as i can tell nothing like this even _exists_ on stable

Reply:
![image](https://github.com/user-attachments/assets/65c05821-9a4f-4aac-83e1-b9987c8af861)

Reply:
The blend mode is multiplicative in stable and it did needs to be here as lot of skins rely on it

Reply:
"Multiplicative" doesn't require anything special. Setting `Colour` on a sprite is multiplicative already.